pypeit/calibrations.py

```diff
@@ -456,7 +444,16 @@ def get_bpm(self):
         sci_image_file = self.fitstbl.frame_paths(self.frame)
 
         # Build it
-        self.msbpm = self.spectrograph.bpm(sci_image_file, self.det)
+        if self.par['makebpm']:
```

Comment:
I think you can reduce this block to one call to `self.spectrograph.bpm` where
you pass in `None` for self.msbias if the bias frame doesn't exist or if the parameter is not set.

Reply:
Done!